tsreport.c

```diff
@@ -476,6 +477,10 @@ static int report_buffering_stats(TS_reader_p  tsreader,
       const int is_discontinuity = (adapt != NULL && (adapt[0] & 0x80) != 0);
       struct stream_data * const ss = stats + index;
 
+      if (printcc)
+        fprint_msg("CC %d\n", cc);
```

Comment:
Is there anything else that could be logged here to make the output more useful (such as the packet id)? Or is it just not relevant? (definitely not claiming to be an expert :stuck_out_tongue:)